synapse/cortex.py

```diff
@@ -5911,10 +5916,16 @@ async def storm(self, text, opts=None):
             proxy = await self._getMirrorProxy(opts)
 
             if proxy is not None:
-                logger.info(f'Offloading Storm query {{{text}}} to mirror.', extra=extra)
+                proxname = proxy._ahainfo.get('name')
+                extra = await self.getLogExtra(mirror=proxname, hash=s_storm.queryhash(text))
+                logger.info(f'Offloading Storm query to mirror {proxname}.', extra=extra)
 
                 mirropts = await self._getMirrorOpts(opts)
 
+                mirropts.setdefault('_loginfo', {})
```

Comment:
Did we name this  with an `_` to mark it as "internal"?

Reply:
no idea. probably? it already exists and I just found it the other day.

Reply:
It is considered internal.